### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Rmano**
This is the first part: make fill color part of the style, as well as scale. 

image
image

### CODE REVIEW

File: doc/circuitikzmanual.tex [hunk 1]
```diff
@@ -491,19 +491,11 @@ \subsection{A more complex tutorial: circuits, Romano style.}
 \end{circuitikz}
 \end{LTXexample}
 
-Another thing I like to modify with respect to the standard is the position of the arrows in transistors, which are normally in the middle the symbol. Using the following settings will move the arrows to the start or end of the corresponding pin.
+Another thing I like to modify with respect to the standard is the position of the arrows in transistors, which are normally in the middle the symbol. Using the following setting (see section~\ref{sec:styling-transistors}) will move the arrows to the start or end of the corresponding pin.
```
Comment (asinghvi17) [inline doc/circuitikzmanual.tex]: doesn't the manual use `fancyref`?

Comment (Rmano) [reply doc/circuitikzmanual.tex]: Dunno ;-) ...

Comment (asinghvi17) [reply doc/circuitikzmanual.tex]: hmm it looks like it uses hyperref but not fancyref.  Nevermind then...one of these days I'll make a PR to replace all refs with Frefs

Comment (Rmano) [reply doc/circuitikzmanual.tex]: Yep. Probably, while at it, we should change the document type from `article` to `book`... with chapters and appendixes --- it's quite big as an article already ;-)

**Comment by Rmano** [comment]
image

**Comment by Rmano** [comment]
image
image

**Comment by Rmano** [comment]
I **think** this is finished. A couple of tests more. but it seems consistent and working.

**Comment by Rmano** [comment]
When merged, this fixes #218

**Comment by Rmano** [comment]
Ok, I tested this (generating a ctan release, and testing it) compiling several things, the manual, and using the bit of context that I know. 
I will merge soon to have the `-git` version testable; I think I'll release coming back from vacation (en of August) to be able to manage the bug reports...

**Pull request merged**